doc/lsst_versions/configuring.rst

```diff
@@ -0,0 +1,37 @@
+Modifying the pyproject.toml file
+=================================
+
+The primary use case for this package is to use it to calculate a version dynamically from the Git repository when it is being built.
+
+This can be enabled by adding a few lines to the ``pyproject.toml`` configuration file for the package.
```

Comment:
If you are not using pyproject.toml then you need to use the `find_dev_lsst_version` API directly and write your own version file -- this is actually how lsst_versions `setup.py` works.

 That was how I was writing my setup.py initially before I learned about the setuptools entry point -- it was painful because for some reason the pip sandbox in the GitHub Action did not have access to all the python packages that were needed (GitPython specifically). This works for this package because `pyproject.toml` has a `build-system` entry that handles those prereqs -- maybe also adding something to `setup.cfg` would fix the problem but I'm not sure it would. Do you want me to see if moving the build-requires information out of pyproject.toml into setup.cfg works?

Do I really need to document how not to use this package as intended?

Reply:
No, as I said, pointing to how to get away from setup.py to pyproject.toml would be fine.